README.md

```diff
@@ -36,6 +36,14 @@ Syncopate is expected to work on any platform vim supports, but the default
 configuration may not work on your system and direct-to-clipboard support hasn't
 been implemented for some platforms. Contributions welcome!
 
+### xclip
+
+Syncopate requires `xclip` to manipulate clipboard. In most cases installing it from your package manager should just work, but for Arch users the `xclip` in official repository(0.12.4) is too old and doesn't support `-target`. They should install `xclip-svn` from AUR:
```

Comment:
Replace this line with the following:

```
Syncopate requires `xclip` to manipulate the clipboard.  In most cases,
installing it from your package manager should just work.

Arch Linux's official repository has an `xclip` (0.12.4) which is too old: it
doesn't support `--target`. Arch users should install `xclip-svn` from AUR:
```

I tweaked the grammar a bit, and wrapped the lines.